core/templates/dev/head/components/summary_tile/exploration_summary_tile_directive.html

```diff
@@ -26,7 +26,7 @@ <h2 class="activity-title protractor-test-exp-summary-tile-title">
             <div ng-repeat="avatar in avatarsList | limitTo:MAX_AVATARS_TO_DISPLAY-1:1">
               <circular-image src="avatar.image"
                               link="avatar.link"
-                              tooltip="<[avatar.tooltipText | truncate:14]>"
+                              tooltip="<[avatar.tooltipText | truncate:40]>"
```

Comment:
Does this actually need to be 40? I thought the only thing it shows is "Community Owned.", but let me know if you found other cases.